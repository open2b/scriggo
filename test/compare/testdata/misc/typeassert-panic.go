// skip

// paniccheck

package main

func main() {
	var i interface{} = 10
	_ = i.(string)
}
