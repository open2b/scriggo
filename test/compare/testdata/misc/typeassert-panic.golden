panic: interface conversion: int is not string